mscore/menus.cpp

```diff
@@ -878,7 +879,7 @@ PalettePanel* MuseScore::newNoteHeadsPalettePanel()
       QAction* action = s->action();
       QIcon icon(action->icon());
       ik->setAction("add-parentheses", icon);
-      sp->append(ik, s->help());
+      sp->append(ik, qApp->translate("Palette", "Add parentheses to notehead"));
```

Comment:
Where does this string stem from? It needs to reference a string marked for translation elsewhere.
I might well have been wrong here earlier and this probably needs to be  `QT_TRANSLATE_NOOP("Palette", "Add parentheses to notehead")` here

Reply:
I just wrote this string here for the first time, so yeah I guess it should be using `QT_TRANSLATE_NOOP`.